packages/ui/src/models/visualization/metadata/beans-entity-handler.ts

```diff
@@ -0,0 +1,187 @@
+import { BeansAwareResource, CamelResource, RouteTemplateBeansAwareResource } from '../../camel';
```

Comment:
Could we add a few tests for this class if possible?

Reply:
Added tests for `RouteTemplateBeansAwareResource` methods into `kamelet-resource.test.ts` :+1:  For `BeansAwareResource` we already have some tests in `camel-route-resource.test.ts`.